README.md

```diff
@@ -47,6 +47,8 @@ protractorFlake({
   nodeBin: 'node',
   // set color to one of the colors available at 'chalk' - https://github.com/chalk/ansi-styles#colors
   color: 'magenta',
+  // set the arguments for protractor
+  // note: the protractor config have to be the first option in protractorArgs
```

Comment:
What about putting this note on `retryConfig` instead of `protractorConfig` because this is only an issue when using `retryConfig`?

Also minor grammar note, I think the sentence would be better reworded as "Arguments passed to protrractor. The path to the protractor config must be the first argument."